### DISCUSSION TITLE
requestName as operationId is not working

### DISCUSSION THREAD

**Discussion by george4joseph** [apideck-libraries/portman#291] [Q&A]
@thim81 
I have used `requestNameSource=Fallback` and the collection generated is not having request name with operationid , its generated with `description` value in openapi.

OPENAPI
```
"/tasks": {
            "get": {
                "operationId": "user.findTask,
                "summary": "task",
                "tags": ["portal_1","portal_2"],
                "description": "task",
                "responses": {
```

Collection generated
```
"item": [
    {
      "id": "8cc803a4-8a2c-4e00-85f1-f0562f55fd84",
      "name": "portal_1",
      "description": {
        "content": "",
        "type": "text/plain"
      },
      "item": [
        {
          "id": "4cd8476f-6fd4-4789-a513-4a70425d9653",
          "name": "task",
          "request": {
            "name": "task",
            "description": {
              "content": "task",
              "type": "text/plain"
 ```

Command
```
portman -l openapi.json -t false -o last.json -O folderStrategy=Tags includeAuthInfoInExample=false requestParametersResolution=Example disableOptionalParameters=true requestNameSource=Fallback
```

It would be helpful if we are able to generate with `name` in collection with `operationId`.

**Comment by thim81** [comment]
hi @george4joseph 

You cannot pass the Postman conversion parameters directly to the CLI.
It is advised to put the Postman conversion variables in a JSON file and pass them like so:

`portman -l openapi.json -t false -o last.json -s .postman-config.json`

Some more context: https://github.com/apideck-libraries/portman#pass-custom-paths-for-config-files
